modules/local/CanonicalAndNoncanonicalProteomes/add_ncrna.nf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Creates the ncRNA protein database
+ */
+process ADD_NCRNA {

```

Comment:
Can you name all the modules with lower case. **the folders I mean**. The process itself can be uppercase.